File: internal/analyze/java/testdata/coupling_no_star.java
package com.acme.billing;

import com.acme.shared.Money;
import com.acme.shared.Ledger;
import static com.acme.shared.Rates.rate;
import java.time.Instant;

class Invoice {
    Money amount;
    Instant at() {
        return Instant.ofEpochMilli(rate() + amount.cents());
    }
}

class Note {
    Ledger l;
}

class Plain {
}

File: internal/analyze/kotlin/testdata/coupling_no_star.kt
package com.acme.billing

import com.acme.shared.Money
import com.acme.shared.Ledger as L
import java.time.Instant

class Invoice(val amount: Money) {    // internal 1, external 1
    fun at(): Instant = Instant.now()
}
class Note {                          // internal 1, external 0
    val l = L()
}
class Plain                           // internal 0, external 0
